### ISSUE TITLE
Characters limit

### ISSUE THREAD

**Issue by fizi** [e107inc/e107#2416]
The limit work but I think not good if limit 200 but the text earlier have the point and limit count stop working, (stop on the point). Would be better if the limit (count of characters ignore the point).

**Comment by rica-carv** [comment]
Err, you're talking about news summary, right?

**Comment by fizi** [comment]
yes, {NEWSSUMMARY: limit=xxx}

**Comment by rica-carv** [comment]
You can change the limit to any number of chars you want.
That shortcode can be eassilly customized in a theme_shortcode.php file, so it wouldn't break sentences, to exclude punctuation, etc.

**Comment by fizi** [comment]
What shortcode need?

**Comment by rica-carv** [comment]
@fizi The {NEWSSUMMARY} (or in fact, any shortcode) can be redefined inside theme_shortcodes file, to work the way intended by the theme designer....

**Comment by Moc** [comment]
Yes, but preferably the first step should be to see if it's already possible to achieve the goal by using core shortcodes and parameters. 

In case this is not possible, the next step is too look for enhancements to the core shortcodes. 

Only after the adjustments have been considered out of scope for the core, then the theme developer should use the theme_shortcodes file to override the core ones. 

@fizi: can you please illustrate your suggestion by giving us an example of how it is currently working and how you would like it to be?

**Comment by fizi** [comment]
I put the image. I set limit=500 (or whatever) after the first point not count the characters more.
![newssummary_limit](https://cloud.githubusercontent.com/assets/86780/22989537/8b1e6fb8-f3b6-11e6-853a-8da7ffb0e2b5.jpg)

**Comment by rica-carv** [comment]
Might be directly related to [line 582](https://github.com/e107inc/e107/blob/f6612410e0c38e41520ec90f52ffe148177b068f/e107_core/shortcodes/batch/news_shortcodes.php#L582)...

Since you don't have a news summary for that news, it goes and fetch the news body to generate that summary.
What i find odd is that you don't have the final "..."
Do you have latest github files?

**Comment by fizi** [comment]
Yes I use latest.

**Comment by CaMer0n** [comment]
{NEWS_SUMMARY: limit=50} Does it work for you @fizi ? If so, please close this issue. thanks.

**Comment by fizi** [comment]
Not work, the problem same.

**Comment by fizi** [comment]
{NEWS_SUMMARY: limit=xxx} Still it does not work, just shows the text by the end of the first sentence.
